src/main/java/com/codeborne/selenide/drivercommands/WebDriverWrapper.java

```diff
@@ -40,10 +40,11 @@ public WebDriver getAndCheckWebDriver() {
     return webDriver;
   }
 
+  /**
+   * Does not close webdriver.
+   * This class holds a webdriver created by user - in this case user is responsible for closing webdriver by himself.
+   */
   @Override
   public void close() {
```

Comment:
I think in this case we should at least log a message that driver wasn't closed automatically and user need additional code to properly close a session